tests/snapshots/TestCLICommands_atmos_describe_config_-f_yaml.stdout.golden

```diff
@@ -43,5 +43,5 @@ default: false
 validate:
     editorconfig:
         color: true
-cli_config_path: /absolute/path/to/repo/examples/demo-stacks/atmos
+cli_config_path: /absolute/path/to/repo/examples/demo-stacks
```

Comment:
When I saw this in the original PR, I thought `atmos` was a directory, and not a file.